### ISSUE TITLE
Remove shadow menu warning

### ISSUE THREAD

**Issue by hadim** [scijava/scijava-common#262]

**Comment by ctrueden** [comment]
I totally agree we need to get rid of the Open Recent warnings. The proper fix is not to stop warning here, but rather to figure out why the Open Recent feature (which has always been half-baked, BTW) causes these warnings.

**Comment by stelfrich** [comment]
> The proper fix is not to stop warning here, but rather to figure out why the Open Recent feature (which has always been half-baked, BTW) causes these warnings.

I have the _feeling_ (not verified) that this has to do with multi-point files which open multiple images with only one filename.
